src/main/java/model/Figure.java

```diff
@@ -0,0 +1,12 @@
+package model;
+
+import utility.AreaCalculator;
+import utility.PrintInfo;
+
+public abstract class Figure implements AreaCalculator, PrintInfo {
```

Comment:
Class names should not be verbs. Consider renaming `AreaCalculator` to something that represents an entity or characteristic, like `AreaCalculable` or `Measurable`.